rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -553,7 +536,7 @@
                 <javaclass references="org.hibernate.id.{idclass}"/>
             </when>
             <perform>
-                <hint title="Class org.hibernate.id.{idclass} was removed/deprecated in Hibernate 5" effort="1" category-id="mandatory">
+                <hint title="Hibernate 5 - Removed deprecated class org.hibernate.id.{idclass}" effort="1" category-id="optional">
```

Comment:
while deprecated methods could be *Optional*, package movements or class removal are not *Optional*.